BondarenkoBE/lab1/ConsoleApplication3/ConsoleApplication3.cpp

```diff
@@ -0,0 +1,46 @@
+// ConsoleApplication3.cpp: ���������� ����� ����� ��� ����������� ����������.
+//
+
+#include "stdafx.h"
+#include <iostream>
+#include <conio.h>
+using namespace std;
+int main()
+{
+	setlocale(0,"");
+	float FirNum, SecNum;
+	char Act;
+    cout << "Input first number: ";
+	cin >> FirNum;
+	cout << "Input action: ";
+	cin >> Act;
+	cout << "Input second number: ";
+	cin >> SecNum;
+	switch (Act) {
+	case '+':
+		cout << FirNum << " " << Act << " " << SecNum << " = "<< FirNum + SecNum;
```

Comment:
At classroom i mentioned that you should store the result in appropriate variable and print it at the end of application **only if** no errors occurred (e.g. zero division)